web/themes/custom/digital_gov/templates/node/node--topics.html.twig

```diff
@@ -123,24 +123,31 @@
     {{ content.field_featured_links|field_value }}
   {% else %}
     {# Only show these sections when on page 1, otherwise hide to show only More News and Events #}
-    {% if is_first_page %}
```

Comment:
I don't think we need this `if` since views should be setting the path for pages beyond the first to a URL that will use a different template. ala <https://digital.gov/topics/challenges-and-prize-competitions/page/3/>